docs/DEVELOPMENT.md

```diff
@@ -0,0 +1,30 @@
+# Development and Testing
+
+## Testing
+
+The `event-manager` is tested using:
+- unit tests - defined in their corresponding modules
+- Rust integration tests - defined in the [tests](../tests) directory
+- performance tests - defined in the [benches](../benches) directory
+
+The integration and performance tests share subscribers implementations
+which can be found under the [src/utilities](../src/utilities) module.
+
+The `utilities` module is compiled only when using the `test_utilities`
+feature. To run unit tests, integration tests, and performance tests, the user
+needs to specify the `test_utilities` feature; otherwise the build fails.
+```bash
```

Comment:
nit: add blank line before code snippets (this and the next)

Reply:
Done.